src/page/Login/Login.jsx

```diff
@@ -0,0 +1,69 @@
+import axios from "axios";
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { Input } from "../../component";
+import { useDocumentTitle } from "../../customHooks";
```

Comment:
You can take a look in to absolute path to avoid ../../
**You can see from here**  [https://dev.to/emanuelone/how-to-use-absolute-path-in-reactjs-503i](url)